dataplane/standalone/switch.cc

```diff
@@ -365,3 +374,40 @@ sai_status_t Switch::set_child_attr(sai_object_type_t type, std::string id,
                                     const sai_attribute_t *attr) {
   return this->apis[id]->set_attribute(attr);
 }
+
+void Switch::handle_notification() {
+  grpc::ClientContext ctx;
+  forwarding::NotifySubscribeRequest req;
+  req.mutable_context()->set_id("default");
```

Comment:
Should this be "DEFAULT"? Also is when the context is created is the name configurable? If so use a common variable?

Reply:
used a variable from go instead and changed name to avoid confusion with network-instance as it's not related